shared/fs/common/tlf-info-line.tsx

```diff
@@ -87,18 +80,34 @@ const getText = (props: Props) => {
   return props.reset ? <ResetText {...props} /> : null
 }
 
-const TlfInfoLine = (props: Props) => (
-  <Kb.Box2
-    direction="horizontal"
-    fullWidth={true}
-    centerChildren={props.mode === 'default'}
-    alignItems="center"
-  >
-    <PrefixText {...props} />
-    <ResetMetaMaybe {...props} />
-    {getText(props)}
-  </Kb.Box2>
-)
+const TlfInfoLine = (props: Props) => {
+  const prefix = <PrefixText {...props} />
+  const dot = (
+    <Kb.Text
+      type="BodySmall"
+      style={props.mode === 'default' ? styles.textDefault : styles.textRow}
+      lineClamp={props.mode === 'row' && Styles.isMobile ? 1 : undefined}
+    >
+      •&nbsp;
+    </Kb.Text>
+  )
+
+  const reset = <ResetMetaMaybe {...props} />
+  const text = getText(props)
+  return (
+    <Kb.Box2
+      direction="horizontal"
+      fullWidth={true}
+      centerChildren={props.mode === 'default'}
+      alignItems="center"
+    >
+      {prefix}
+      {prefix && (reset || text) ? dot : null}
```

Comment:
I still see some of these
<img width="236" alt="image" src="https://user-images.githubusercontent.com/11968340/64349704-64504580-cfc5-11e9-961c-22dc6c376db1.png">

Reply:
LMK if I can help debug

Reply:
Yeah cause I was never able to repro in the first place

Reply:
to reproduce maybe just set this to 0? https://github.com/keybase/client/blob/938410b53d87b0d651718e9e63f4936e09b1d3c4/shared/actions/fs/index.tsx#L108

Reply:
ok so the issue is the siblings i'm checking always exist, even though inside they render to 0. they don't really serve a purpose being actual components so i just made them functions so we get either null or some other component and this logic works